packages/api/drizzle.config.ts

```
@@ -0,0 +1,10 @@
+import { defineConfig } from 'drizzle-kit'
+
+export default defineConfig({
+    dialect: 'postgresql',
+    schema: './src/db/schema.ts',
+    out: './migrations',
+    dbCredentials: {
+        url: process.env.DATABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)